nemo_curator/datasets/doc_dataset.py

```diff
@@ -65,10 +66,10 @@ def read_json(
     @classmethod
     def read_parquet(
         cls,
-        input_files,
-        backend="pandas",
-        files_per_partition=1,
-        add_filename=False,
+        input_files: Union[str, List[str]],
+        backend: str = "pandas",
+        files_per_partition: int = 1,
+        add_filename: bool = False,
```

Comment:
Thanks a lot for adding annotations here. Could we also add some docstrings here as well?